z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-bootstrap/tasks/main.yaml

```diff
@@ -76,4 +75,33 @@
     volume_size: "{{ bootstrap_flavor_size.stdout_lines[0]}}"
     terminate_volume: True
   when:
-  - disk_type == "scsi"
\ No newline at end of file
+  - disk_type == "scsi"
+  - use_storage_provider == false
+
+- name: 'Create BFV bootstrap volume from backend storage provider'
+  os_volume:
+    state: present
+    name: "{{ os_bootstrap_server_name }}-bfv"
```

Comment:
1. from doc: https://docs.ansible.com/ansible/2.4/os_volume_module.html
I only see display_name, is "name" also a supported parameter for the os_volume module?

2. change -bfv to -root?

Reply:
yes, already changed.